tests/backends/sqlite/test_introspection.py

```diff
@@ -25,3 +27,218 @@ def test_get_primary_key_column(self):
                         self.assertEqual(field, expected_string)
                     finally:
                         cursor.execute('DROP TABLE test_primary')
+
+    def _parse_column_or_constraint_definition(self, sql, columns):
+        statement = sqlparse.parse(sql)[0]
+        tokens = (token for token in statement.flatten() if not token.is_whitespace)
+        with connection.cursor() as cursor:
+            return connection.introspection._parse_column_or_constraint_definition(tokens, columns)
+
+    def test_unique_column_parsing(self):
+        for sql in [
```

Comment:
Perhaps it would be less verbose to consolidate more cases into a single test with something like:
```
tests = (
    ('"ref" integer UNIQUE,', ['ref']),
   ...
)
for sql, cols in tests:
    with subTest(sql=sql):
        ...

```
Use`self.subTest()` in the loop.

Reply:
changed, now it looking match more readable, thanks